wcfsetup/install/files/lib/system/box/UserOnlineListBoxController.class.php

```diff
@@ -65,6 +85,12 @@ protected function readObjects()
     {
         EventHandler::getInstance()->fireAction($this, 'readObjects');
 
+        if ($this->sortOrder && $this->sortField === 'lastActivityTime') {
```

Comment:
I think it would be useful to compare the sortfield using `in_array` against an 1-element array or to convert this into a `switch()`, to allow for cleaner extensions in case other fields are of interest.

In fact: Don't you also need to check for `requestURI` here?

Reply:
Changes to this code are unlikely as it is basically the same as in the controller, see https://github.com/WoltLab/WCF/blob/be96b55eea61c55177878d7454e396f634169773/wcfsetup/install/files/lib/page/UsersOnlineListPage.class.php#L143.

Reply:
Oh, the `lastActivityTime` needs special-casing because it exists in both the user as well as the session table. This is dumb.